sdk/tables/azure-data-tables/azure/data/tables/_deserialize.py

```diff
@@ -46,18 +46,15 @@ def get_enum_value(value):
         return value
 
 
-def _from_entity_binary(value):
-    # type: (str) -> EntityProperty
+def _from_entity_binary(value: str) -> EntityProperty:
```

Comment:
I'm not sure if this is actually returning an EntityProperty?

Reply:
It should be `bytes`